src/chart.zoom.js

```diff
@@ -19,185 +19,222 @@ var panFunctions = zoomNS.panFunctions = zoomNS.panFunctions || {};
 
 // Default options if none are provided
 var defaultOptions = zoomNS.defaults = {
-	pan: {
-		enabled: true,
-		mode: 'xy',
-        speed: 20,
-		threshold: 10,
-	},
-	zoom: {
-		enabled: true,
-		mode: 'xy',
-	}
+        pan: {
+                enabled: true,
+                mode: 'xy',
+                speed: 20,
+                threshold: 10,
+        },
+        zoom: {
+                enabled: true,
+                mode: 'xy',
+                sensitivity: 3,
+        }
 };
 
 function directionEnabled(mode, dir) {
-	if (mode === undefined) {
-		return true;
-	} else if (typeof mode === 'string') {
-		return mode.indexOf(dir) !== -1;
-	}
+        if (mode === undefined) {
+                return true;
+        } else if (typeof mode === 'string') {
+                return mode.indexOf(dir) !== -1;
+        }
 
-	return false;
+        return false;
 }
 
-function zoomIndexScale(scale, zoom, center) {
-
+function zoomIndexScale(scale, zoom, center, zoomOptions) {
+        var labels = scale.chart.data.labels;
+        var minIndex = scale.minIndex;
+        var lastLabelIndex = labels.length - 1;
+        var maxIndex = scale.maxIndex;
+        var sensitivity = zoomOptions.sensitivity;
+
+        zoomNS.zoomCumulativeDelta = zoom > 1 ? zoomNS.zoomCumulativeDelta + 1 : zoomNS.zoomCumulativeDelta - 1;
+
+        if (Math.abs(zoomNS.zoomCumulativeDelta) > sensitivity){
+                if(zoomNS.zoomCumulativeDelta < 0){
+                        if(center.x <= scale.width/2){
```

Comment:
I was trying to not leave a dead zone when the pointer/pinch centre was at dead center as the other check is just for greater than.
Hadn't thought about minification, but I will now!
I'll fix the isHorizontal issue too
The indents are still bugging me too but I didn't have time to sort then out. Next commit will hopefully. Do you prefer tab or space indents?